components/supervisor/hot-deploy.sh

```diff
@@ -18,9 +18,9 @@ dev_image="$(tar xfO "$bldfn" ./imgnames.txt | head -n1)"
 echo "Dev Image: $dev_image"
 
 
-cf_patch=$(kubectl get cm server-ide-config -o=json | jq '.data."config.json"' |jq -r)
+cf_patch=$(kubectl get cm ide-config -o=json | jq '.data."config.json"' |jq -r)
 cf_patch=$(echo "$cf_patch" |jq ".supervisorImage = \"$dev_image\"")
 cf_patch=$(echo "$cf_patch" |jq tostring)
 cf_patch="{\"data\": {\"config.json\": $cf_patch}}"
 
-kubectl patch cm server-ide-config --type=merge -p "$cf_patch"
+kubectl patch cm ide-config --type=merge -p "$cf_patch"
```

Comment:
it would be good if we can restart `ide-service` to get this configmap change immediately, see [doc](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically)
i.e. `kubectl rollout restart deployment ide-service`